### PR TITLE
add landing message #1

### PR DESCRIPTION
**Pull request by Lex-Locke7**
Added <h1> tags to 1st line of file. Ready to be reviewed. -Lex

### CODE REVIEW

**Comment by woodssx** [comment]
I am not convinced about the <h1> code.

**Review by Lex-Locke7 — COMMENTED** [review]
Awaiting review. -Lex

**Review by woodssx — COMMENTED** [review]
Review is complete.

File: index.html [hunk 1]
```diff
@@ -1 +1,2 @@
-This is used to host the code for the new SaaS project.
+<h1>This is used to host the code for the new SaaS project.</h1>
```
Comment (woodssx) [inline index.html:1]: The review is complete.

**Review by woodssx — COMMENTED** [review]
Complete

**Review by woodssx — APPROVED** [review]

**Pull request merged**